manim/utils/color.py

```diff
@@ -294,46 +350,45 @@ def color_to_rgb(color):
         raise ValueError("Invalid color type")
 
 
-def color_to_rgba(color, alpha=1):
+def color_to_rgba(color: Union[Color, str], alpha: float = 1) -> np.ndarray:
     return np.array([*color_to_rgb(color), alpha])
 
 
-def rgb_to_color(rgb):
-    try:
-        return Color(rgb=rgb)
-    except Exception:
-        return Color(WHITE)
+def rgb_to_color(rgb: Iterable[float]) -> Color:
+    return Color(rgb=rgb)
 
 
-def rgba_to_color(rgba):
+def rgba_to_color(rgba: Iterable[float]) -> Color:
```

Comment:
Why Iterable and not sequence?

Reply:
I because I thought it was specific enough. Is it not?

Reply:
I remember that I looked up once that Sequence is a bit more specfic then Iterable, but I can not find the link anymore. As it does not make a huge difference, we can stay with Iterable here :-)